### ISSUE TITLE
New exception request: No Attribution for Binary distributions

### ISSUE THREAD

**Issue by zvr** [spdx/license-list-XML#2282]
### How license meets inclusion principles

A simple and hopefully clear exception to a standard permissive license (used with MIT) that specifies that there is no need for reproducing copyright and license texts if the code is compiled and included in a binary artifact.

### License Name

No Attribution for Binary distributions

### Suggested short identifier

NoAttribBinary

### License or Exception?

exception

### URL to license text

https://github.com/fmtlib/fmt/blob/master/LICENSE

### OSI Status

Not Submitted

### License author or steward

Victor Zverovich

### URL to project(s) that use license

https://github.com/fmtlib/fmt

### paste text of license here

As an exception, if, as a result of your compiling your source code, portions of this Software are embedded into a machine-executable object form of such source code, you may redistribute such embedded portions in such object form without including the above copyright and permission notices.

**Comment by richardfontana** [comment]
This package is found in major distros such as Fedora.

**Comment by jlovejoy** [comment]
@richardfontana - is there a Fedora issue related to this, by any chance?

+1 to add this. I think this may be the first exception to MIT we've seen! At first I was thinking that it would be a new license as it's all together, but it is very explicit as to "Optional exception to the license" which seems to make `MIT WITH <name>` make sense

Perhaps we use "exception" in the id to be consistent?
maybe `NoBinaryAttrib-exception` or the order @zvr had with -exception added??

**Comment by richardfontana** [comment]
@jlovejoy, no, I just verified that the package is in Fedora after seeing this issue.

**Comment by InSwiss** [comment]
+1 to add. We think it can be added as 'MIT WITH No Binary Attribution'  as it is being more permissive to an already permissive license we don't think word 'exception' is relevant.

**Comment by swinslow** [comment]
+1 to add. From a Google search I am seeing this exception showing up in many projects' / products' license notice disclosure lists.

For name and ID, we typically include the word "exception" if it is clearly an exception to an obligation. (See https://spdx.org/licenses/exceptions-index.html with many examples.) I would say that's the case here, since it gives a permission not to do something that would otherwise be required by the license.

Often we have used the name of a project as the exception name, if there's a particular project that it's most notably tied to?

Most of the instances where I'm seeing it used appear to be tied to this `fmt` library that @zvr originally filed this for, with a copyright date starting in 2012. E.g.:
* https://ccache.dev/license.html
* https://www.resolvebim.com/open-source-acknowledgements
* https://docs.nvidia.com/nsight-dl-designer/2021.1/CopyrightAndLicenses/index.html
* https://learn.foundry.com/katana/dev-guide/Licenses/index.html
as just a few examples.

However, I do see one instance that is earlier: https://docs.oracle.com/en/database/oracle/developer-tools-for-vscode/21.7.1/licensing-guide/index.html shows this exception used for "Jb Evain" with a copyright date of 2006. Might be worth seeing if that seems to be the original source for this text?

**Comment by swinslow** [comment]
Taking a bit of a closer look at the "Jb Evain" item: This appears to originate from [dotnet/runtime](https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT). And, digging into that particular file and the Git history, I think the "Jb Evain" notice actually got dropped into the middle of the "fmt" notice just above it -- accidentally cutting off the exception. All of which is to say, I think we can disregard that one.

So, the only place where I'm seeing this particular exception used is with the "fmt" library.

Given that, can we go with something like `fmt-exception` or `fmt-MIT-exception` for the ID, and "fmt MIT Exception" for the name?

**Comment by jlovejoy** [comment]
# License Inclusion Decision

## Decision:

- [ ] approved
- [ ] not approved

## Name
fmt exception
## License ID
fmt-exception
## XML markup
none
## Notes:
Typically used with MIT.
## Next steps ##
If the license has been accepted, please follow [the accepted-license process to create the PR](https://github.com/spdx/license-list-XML/blob/main/DOCS/request-new-license.md#accepted-license-process).

**Comment by jlovejoy** [comment]
@zvr - assuming you are ok with name and ID, do you want to make a PR for this?
